### ISSUE TITLE
setup:di:compile error

### ISSUE THREAD

**Issue by kinhcan** [omise/omise-magento#115]
I got problem when run
`php bin/magento setup:di:compile at version 2.1.6 and 2.2.x`
error: 
```
[ReflectionException]                            
  Class Omise\Payment\Model\Config does not exist 
```
I did not found class Model/config at code folder.

**Comment by guzzilar** [comment]
@kinhcan this issue has been solved in https://github.com/omise/omise-magento/pull/106. However, this fix hasn't been released yet.
Would you mind to do workaround for now, just delete the following files out from the plugin folder (the same as what I have fixed in that pull request):
- Model/Ui/OmiseConfigProvider.php
- Test/Unit/Model/ConfigTest.php
- Test/Unit/Model/Ui/OmiseConfigProviderTest.php

Then, execute the command `php bin/magento setup:di:compile` again.

Sorry for the inconvenience,
and thanks for submitting the report 🙂

**Comment by kinhcan** [comment]
Thank for your solution, I fixed this issue.

**Comment by guzzilar** [comment]
You're welcome :)

**Comment by tamada1** [comment]
So, when will it be released?

**Comment by jacekstan** [comment]
we have fixed in recent release 2.3, so I am closing that issue.
